Sources/Clock2.swift

```diff
@@ -55,6 +55,15 @@ public enum Clock2 {
         latestOffset = nil
         protection.unlock()
     }
+    
+    /// The most recent stored ntp date. Getting retrieves from the UserDefaults defined by the storage policy.
+    public static var storedNow: Date? {
```

Comment:
Is this safe to be called from multiple threads concurrently?

Reply:
Yes, I believe so. Both the `stableTime` variable in `TimeStorage` and `adjustedTimestamp` in `TimeFreeze` should be safe to access concurrently after my changes.